dist/bin/no-internal-report.js

```diff
@@ -1,64 +1,44 @@
 #! /usr/bin/env node
-
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 "use strict"
 const path = require('path');
-const fs = require('fs');
-
-function getNodeModulesDir(startPath) {
-  let currPath = startPath;
-  let baseName = path.basename(currPath);
-  while (baseName !== "node_modules") {
-    currPath = path.resolve(currPath, "..")
-    if (currPath === "/")
-      return undefined;
-    baseName = path.basename(currPath);
+const assert = require('assert');
+const eslintBinPath = path.join(require.resolve("eslint"), "../../bin/eslint.js");
+
+const args = { tags: ["alpha", "internal"], files: [] };
+for (let i = 2; i < process.argv.length; ++ i) {
+  if (process.argv[i] == '--tags') {
+    assert(i + 1 < process.argv.length, "tags option requires an argument");
+    args.tags = process.argv[i + 1].split(",");
+    i += 1; // skip next arg
+  } else if(process.argv[i] == '--files') {
```

Comment:
Maybe not the right place to ask this and should be a separate issue here but are multiple file globs not allowed as arguments?

Reply:
changes made allow multiple file globs